.eslintrc.yml

```diff
@@ -13,6 +13,7 @@ globals:
   before: true
   after: true
   beforeEach: true
+  afterEach: true
```

Comment:
could you explain this, please?

Reply:
because eslintrc shows the error "afterEach is not defined", and for this case i defined that here.